### ISSUE TITLE
Fermi support is broken

### ISSUE THREAD

**Issue by maddyscientist** [lattice/quda#507]
It appears that dslash_test is broken when running on Fermi:

```
warning: Cuda API error detected: cudaBindTexture returned (0x1e)

warning: Cuda API error detected: cudaGetLastError returned (0x1e)

ERROR: (CUDA) unknown error (/home/kate/github/quda-develop/lib/dslash_wilson.cu:163 in wilsonDslashCuda())
       last kernel called was (name=N4quda16WilsonDslashCudaI6float4S1_EE,volume=12x24x24x24,aux=type=single-GPU,reconstruct=18)
```

I suspect the problem is related to some of the changes made with respect to peer-to-peer, but this will require checking.

**Comment by maddyscientist** [comment]
Closed with 8073b851627e90133d04ed94a6e5b1fcb05045f5.

@weinbe2 this should unblock you from running on Fermi.

**Comment by mathiaswagner** [comment]
Fix is in develop
